WinImageBuilder.psm1

```diff
@@ -361,14 +361,25 @@ function Download-CloudbaseInit {
         [Parameter(Mandatory=$true)]
         [string]$resourcesDir,
         [Parameter(Mandatory=$true)]
-        [string]$osArch
+        [string]$osArch,
+        [parameter(Mandatory=$false)]
+        [switch]$betaRelease=$true
+
     )
     Write-Host "Downloading Cloudbase-Init..."
 
-    if ($osArch -eq "AMD64") {
-        $CloudbaseInitMsi = "CloudbaseInitSetup_Stable_x64.msi"
+    if ($betaRelease){
+        if ($osArch -eq "AMD64") {

```

Comment:
use a variable and then -format to keep the code clean and without code duplicates.